boot/boot_serial/src/boot_serial.c

```diff
@@ -588,7 +588,7 @@ boot_serial_in_dec(char *in, int inlen, char *out, int *out_off, int maxout)
         if (len != *out_off - sizeof(uint16_t)) {
             return 0;
         }
-        len = min(len, *out_off - sizeof(uint16_t));
+        len = MIN(len, *out_off - sizeof(uint16_t));
```

Comment:
Please only do this for Zephyr, Mynewt uses `min`.

Reply:
We should either define these ourselves (which is less trivial than it seems), or define our own wrapper that uses the appropriate one for the system. Or, my suggestion, since this only occurs once in the code, just change this to:
```c
    if (len > *out_off - sizeof(uint16_t)) {
        len = *out_off - sizeof(uint16_t);
    }
```

Reply:
I did as you suggest.